build/hygiene.js

```diff
@@ -195,7 +195,7 @@ function createGitIndexVinyls(paths) {
 				}
 
 				cp.exec(
-					`git show :${relativePath}`,
+					process.platform === 'win32' ? `git show :"${relativePath}"` : `git show ':${relativePath}'`, // {{SQL CARBON EDIT}} Add double quotes around Windows path
```

Comment:
any idea why they don't need this upstream in vscode repo?

Reply:
No idea, but I couldn't check in that javascript regex file without it. Going by the history, this whole part used to be single quoted, but then was changed to have no quotes on windows to fix some other kind of issue. The javascript regex file hasn't changed in 6 years, so maybe they haven't hit issues with any other filenames with spaces. That's certainly the only one I've seen while working on ADS the past few years.

Reply:
If you want, I could just revert this file. The only thing that was being blocked was comitting that one javascript file.

Reply:
Yeah let's do that - if it comes up again we can look into it further but it seems extremely unlikely to happen in most cases